fixposition_odometry_converter_ros2/include/fixposition_odometry_converter_ros2/odom_converter_node.hpp

```diff
@@ -0,0 +1,89 @@
+/**
+ *  @file
+ *  @brief Declaration of OdomConverterNode class
+ *
+ * \verbatim
+ *  ___    ___
+ *  \  \  /  /
+ *   \  \/  /   Fixposition AG
+ *   /  /\  \   All right reserved.
+ *  /__/  \__\
+ *
+ * Port to ROS 2 by Husarion
+ * \endverbatim
+ *
+ */
+
+#ifndef __ODOM_CONVERTER_HPP__
+#define __ODOM_CONVERTER_HPP__
```

Comment:
```suggestion
#ifndef __ODOM_CONVERTER_NODE_HPP__
#define __ODOM_CONVERTER_NODE_HPP__
```

Just to keep the include guard the same as the file name